cluster/charts/universal-crossplane/values.yaml.tmpl

```diff
@@ -40,16 +37,18 @@ configuration:
 
 imagePullSecrets: {}
 
-registryCaBundleConfig: {}
+registryCaBundleConfig:
+  key: {}
+  name: {}
```

Comment:
Syncing to the latest upstream: https://github.com/crossplane/crossplane/commit/ca6d692a2678fe557d86ee8cfc6940d8554d3d4e#diff-e1f3e0d8f986aa70ed413b67ab22654989f76829f687338fa246ef55f2a3059bR51-R55